pyproject.toml

```diff
@@ -1,7 +1,61 @@
+[tool.poetry]
+name = "langchain"
+version = "0.0.26"
```

Comment:
Thoughts on deferring this redundancy in this PR? I can look into a fast follow for setting up a release pipeline with poetry.